skorch/callbacks/logging.py

```diff
@@ -205,6 +207,82 @@ def on_train_end(self, net, **kwargs):
         if self.close_after_train:
             self.experiment.stop()
 
+class WandbLogger(Callback):
+    """Logs best model and metrics to `Weights & Biases <https://docs.wandb.com/>`_
+
+    Use this callback to automatically log best trained model, all metrics from
+    your net's history, model topology and computer resources to Weights & Biases
+    after each epoch.
+
+    See `example run
+    <https://app.wandb.ai/borisd13/skorch/runs/s20or4ct/overview?workspace=user-borisd13>`_
+
+    Examples
+    --------
+    >>> # Install wandb
+    ... pip install wandb
+
+    >>> import wandb
+    >>> from skorch.callbacks import WandbLogger
+
+    >>> # Create a wandb Run
+    ... wandb_run = wandb.init()
+    >>> # Log hyper-parameters (optional)
+    ... wandb.config.update({"learning rate": 1e-3, "batch size": 32})
+
+    >>> net = NeuralNet(..., callbacks=[WandbLogger(wandb_run)])
+    >>> net.fit(X, y)
+
+    Parameters
+    ----------
+    wandb_run : wandb.wandb_run.Run
+      wandb Run used to log data.
+
+    save_model : bool (default=True)
+      Whether to save a checkpoint of the best model.
```

Comment:
Maybe add an explanation here that the model will be uploaded to the W&B server for convenience.